browser/src/UI/components/Tabs.tsx

```diff
@@ -235,6 +235,10 @@ export const shouldShowFileIcon = (state: State.IState): boolean => {
 }
 
 export const checkTabBuffers = (buffersInTabs: number[], buffers: State.IBuffer[]): boolean => {
+    if (buffersInTabs instanceof Array === false || buffers instanceof Array === false) {
```

Comment:
Following from the discussion - I wonder if it makes sense to push this 'upward' to here:
https://github.com/onivim/oni/blob/44e162d359c6b2a669e5d6dbcef4589c4de90b56/browser/src/Editor/NeovimEditor/NeovimEditor.tsx#L456

Here, we can check if `response[0][index]` is a valid array, and if not, use an empty array. That way, everything 'downstream' using the `State.ITabState` typing can rely on the type system - that `tab.buffersInTab` is an actual array of numbers. Otherwise, we might end up having to duplicate this check in other places we use `tab.buffersInTab`.